### PR TITLE
Cleanup duplicate colors

### PR DESCRIPTION
**Pull request by nathanbowser**
This is a breaking change! And I'll do the legwork in the other projects but I'd love to have all eyes on this.

### CODE REVIEW

File: scoreboard-colors.scss [hunk 1]
```diff
@@ -113,104 +113,108 @@ $color-badge-disabled-border: rgba($color-badge-border, .5);
...
+  gray $palette-gray,
+  slate $palette-slate,
+  infantry-blue $palette-infantry-blue,
+  white $palette-white
+
+  initiative-budget $palette-initiative-budget,
+  initiative-schedule $palette-initiative-schedule,
+  lighter-gray $palette-lighter-gray;
 
 // These aren't used anywhere yet. They're for colored cells where the text is a darker
```
Comment (nathanbowser) [inline scoreboard-colors.scss:219]: I asked a comment in slack regarding this comment the commented out code below it. It's from 2018 and I think it causes more confusion than it's worth. It's pretty dated, can we delete the commented out code? @scottoreilly You wrote this block.

Comment (scottoreilly) [reply scoreboard-colors.scss:219]: Absolutely, let’s remove commented out stuff.

File: scoreboard-colors.scss [hunk 2]
```diff
@@ -399,3 +293,11 @@ $color-tree-editable-highlight: $color-gold;
     }
   }
 }
+
+// Expose the all the palette colors as css3 variables. Used by JS components
+:root {
+  --palette-colors: #{$palette-colors};
```
Comment (derekstory) [inline scoreboard-colors.scss:299]: I don't think I've seen this done before. Not entirely sure how it works. It stores the entire list as a variable?

Comment (nathanbowser) [reply scoreboard-colors.scss:299]: Yea, it stores it as key/values which I then read in JS. 
Screenshot 2023-02-28 at 2 18 30 PM

Comment (derekstory) [reply scoreboard-colors.scss:299]: Ah, the output was different than I thought it would be. Thanks for the explanation!

**Review by derekstory — APPROVED** [review]
+1

**Comment by jerryorr** [comment]
+1

**Comment by mattsgarlata** [comment]
👍

**Pull request merged**